client/src/Components/AdminEditGroup.js

```diff
@@ -3,6 +3,13 @@ import css from './Admin.module.css'
 import DragNDrop from './DNDComponents/DragNDrop.js';
 
 class AdminEditGroup extends Component{
+
+    backtoproject()
```

Comment:
@Yunwei-Zhang  would you be able to use snakecase or camel case? so `backToProject` instead of `backtoproject`? Please apply the similar formatting for all other files in the React app

Reply:
yea, I will change it then